pymongo/monitoring.py

```diff
@@ -914,13 +914,12 @@ class ConnectionCheckOutFailedReason(object):
 
 
 class _ConnectionEvent(object):
```

Comment:
I refactored the private base class hierarchy here to make _ConnectionEvent the base class for ALL connection events_ConnectionEvent the base class for ALL connection events. This makes the event filtering for CMAP events easier since we can do `isinstance(e, (_ConnectionEvent, _PoolEvent))`